# This is a message to hold racing data:
int16 LapNumber
int16 PlannerCounter
